### ISSUE TITLE
sample C++ application

### ISSUE THREAD

**Issue by zaki50** [KLab/PlaygroundOSS#41]
Is there a sample C++ application?

**Comment by muojp** [comment]
No, nothing exists at the moment. C# -> C++ compiled version would be something very close to your need, I suppose.

**Comment by LysanderGG** [comment]
Actually, there is a basic sample for C++ which is the C++ compiled version of https://github.com/KLab/PlaygroundOSS/tree/master/SampleProject/CSharp

Take a look at https://github.com/KLab/PlaygroundOSS/blob/master/Doc/Documentation_CSharp.md#how-to-run-using-c-converted-to-c to see how to run it.

**Comment by zaki50** [comment]
Thanks!!
